client/src/components/History/Content/ContentItem.vue

```diff
@@ -123,6 +123,7 @@ import { JobStateSummary } from "./Collection/JobStateSummary";
 import { library } from "@fortawesome/fontawesome-svg-core";
 import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
 import { faArrowCircleUp, faMinusCircle, faCheckCircle } from "@fortawesome/free-solid-svg-icons";
+import { useEntryPointStore } from "../../../stores/entryPointStore";
```

Comment:
```suggestion
import { useEntryPointStore } from "stores/entryPointStore";
```

For stores and other things that live at a defined location in the project (esp. when far away from the importer) it's cleaner to use non-relative imports.  The alias isn't in this branch, but when this is refactored this will be easier to keep consistent with `@/stores/entryPointStore` and so on.